jsonschema-validation.xml

```diff
@@ -1098,6 +1098,66 @@
         </section>
 
         <section title="Schema annotations and extension vocabularies">
+            <t>
+                Schema validation is a useful mechanism for annotating instance data
+                with additional information.  This section describes the rules for
+                determining when and how annotations apply to an instance, and specifies
+                a small general-purpose annotation vocabulary.
+            </t>
+            <t>
+                Additional vocabularies SHOULD make use of this mechanism for applying
+                their keywords to instances.
+            </t>
+            <section title="Applicability and attachment">
+                <t>
+                    Annotations can be applied to an instance when it is valid against
+                    the schema that includes the annotation keywords.  For any given location
+                    in the instance (referred to as a sub-instance), each (sub)schema against
+                    which it successfully validates is considered to be <spanx>applicable</spanx>
+                    to that sub-instance.  The (sub)schema is said to be <spanx>attached</spanx>
+                    to each sub-instance to which it applies.
+                </t>
+                <t>
+                    A validation implementation MAY choose to implement determining subschema
+                    applicability and providing access to the value(s) of applicable annotation
+                    kewords.  Implementation of this feature MAY instead be done separately.
+                </t>
+                <t>
+                    Since many subschemas can be applicable to any single sub-isntance, each
```

Comment:
typo: isntance

Reply:
@handrews That one is still there.

Reply:
dammit... I distinctly remember searching for this, but I must have done it in hyperschema and forgotten about validation.  Fixed now.  For reals.  I think.